internal/commands/config/trusted_builder.go

```diff
@@ -0,0 +1,113 @@
+package config
+
+import (
+	"sort"
+
+	"github.com/pkg/errors"
+
+	"github.com/buildpacks/pack/internal/commands"
+	"github.com/buildpacks/pack/internal/style"
+
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+func trustedBuilder(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "trusted-builder",
+		Short:   "Interact with trusted builders",
+		Aliases: []string{"trusted-builders"},
```

Comment:
```suggestion
		Use:     "trusted-builders",
		Short:   "Interact with trusted builders",
		Aliases: []string{"trusted-builder"},
```

The primary option should be plural IMO (as based on RFC).

Reply:
I had totally missed that, you're 100% right. I'll fix that, and all occurences